tests/release/test_snippets-2.go

```diff
@@ -0,0 +1,96 @@
+package release
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/redhat-appstudio/e2e-tests/pkg/framework"
+	gitopsv1alpha1 "github.com/redhat-appstudio/managed-gitops/appstudio-shared/apis/appstudio.redhat.com/v1alpha1"
+)
+
+var snapshotComponents = []gitopsv1alpha1.ApplicationSnapshotComponent{
+	{"component-1", "quay.io/redhat-appstudio/component1@sha256:d5e85e49c89df42b221d972f5b96c6507a8124717a6e42e83fd3caae1031d514"},
+	{"component-2", "quay.io/redhat-appstudio/component2@sha256:a01dfd18cf8ca8b68770b09a9b6af0fd7c6d1f8644c7ab97f0e06c34dfc5860e"},
+	{"component-3", "quay.io/redhat-appstudio/component3@sha256:d90a0a33e4c5a1daf5877f8dd989a570bfae4f94211a8143599245e503775b1f"},
+}
+
+var _ = framework.ReleaseSuiteDescribe("test-release-service-happy-path", Label("release"), func() {
```

Comment:
Hi @naftalysh, can you please update the describe to something like this https://github.com/redhat-appstudio/e2e-tests/blob/main/tests/e2e-demos/config/default.yaml#L2 ? I have added similar note in contribution in this PR: https://github.com/redhat-appstudio/e2e-tests/pull/209 . Thank you!

Reply:
Hi @jkopriva 
Updated as requested